config.toml

```diff
@@ -35,21 +38,22 @@ pluralizeListTitles = false
   participation_level = "participation_levels"
 
 [Author]
-  name = "Christopher Guindon"
+  name = "OSGi Working Group"
   website = "https://www.eclipse.org"
   email = "webdev@eclipse-foundation.org"
-  facebook = "eclipse.org"
-  twitter = "EclipseFdn"
-  youtube = "EclipseFdn"
-  googleplus = "+Eclipse"
-  linkedin = "company/eclipse-foundation/"
+  twitter = "OSGiAlliance"
+  linkedin = "groups/122461/"
 
 [permalinks]
   news = "/:sections/:year/:month/:day/:slug/"
 
-[blackfriday]
-  plainIDAnchors = true
-  hrefTargetBlank = true
+[markup]
```

Comment:
I agree that the blackfriday block can be removed here (our minimum supported version of Hugo is such that Goldmark is always the default), but we don't need to set the default as goldmark in this case as its already the default and we don't override it in our theme. 

By default, it seems that all of the extensions that are [enabled by GFM](https://github.com/yuin/goldmark/#built-in-extensions) should [already be enabled by default](https://gohugo.io/getting-started/configuration-markup/#goldmark). Does it change the rendering when its included?

Reply:
I don't know. I needed to get some html support for super/sub scripting in some text for math formulas. And Hugo is new to me and somewhat impenetrable since I don't know what the base setup is :-).

Reply:
Honestly, I fully get that. When I started using it in May of 2019 I was in the same boat, and still am for some of the weirder things! You can probably remove the line for default markup and leave the `unsafe` property and you should be fine. The `unsafe` isn't recommended, but you have a use case with tags not supported by markdown (sup/sub) so it's one of those cases where it's fine :)

As an aside, if you run into anything really stumping you, feel free to reach out to us with questions. We can't guarantee how fast we'll get back to you, but it shouldn't be long.

Reply:
I made this change.